mne/decoding/time_gen.py

```diff
@@ -390,11 +446,13 @@ def _predict_time_loop(X, estimators, cv, slices, predict_mode,
     ----------
     X : ndarray, shape (n_epochs, n_features, n_times)
         To-be-fitted data.
-    estimators : array-like, shape (n_times, n_folds)
-        Array of scikit-learn classifiers fitted in cross-validation.
-    slices : list
-        List of slices selecting data from X from which is prediction is
-        generated.
+    estimators : list of array-like, shape (n_times, n_folds)
+        List of array of scikit-learn classifiers fitted in cross-validation.
```

Comment:
That if at all only is a drawback for people who really expect it to behave like sklearn, right? Because it's not inherently more complex, it's just that it works against sklearn conventions.

And I think adhering to MNE conventions (where everything takes MNE objects, not arrays) should trump adhering to sklearn conventions.